src/test/java/bio/terra/stairway/fixtures/TestUtil.java

```diff
@@ -106,17 +130,27 @@ private static Stairway makeStairway(
 
   // For cases where we want to validate recovery of a single stairway instance and that
   // that a specific flight is READY and unowned.
-  public static Stairway makeStairwayValidateRecovery(String stairwayName, String flightId)
+  public static Stairway makeStairwayValidateRecovery(
+      String stairwayName, String flightId, int hooks)
       throws DatabaseOperationException, QueueException, MigrateException,
           StairwayExecutionException, InterruptedException, DatabaseSetupException {
     DataSource dataSource = makeDataSource();
-    Stairway stairway =
+
+    Stairway.Builder builder =
         Stairway.newBuilder()
             .stairwayClusterName("stairway-cluster")
             .stairwayName(stairwayName)
             .workQueueProjectId(null)
-            .maxParallelFlights(2)
-            .build();
+            .maxParallelFlights(2);
+
+    for (int i = 0; i < hooks; i++) {
```

Comment:
Not sure if this is better or worse but you can also write this using java streams, if you wanted to pass a list to the builder directly.
```
        List<TestHook> hooks = 
                IntStream.range(0, numHooks)
                .map(x -> x + 1)
                .mapToObj(String::valueOf)
                .map(TestHook::new)
                .collect(Collectors.toList());
```